templates/cli/lib/commands/deploy.js.twig

```diff
@@ -181,7 +181,7 @@ const deployFunction = async () => {
                     name: func.name,
                     runtime: func.runtime,
                     execute: func.execute,
-                    vars: func.vars,
+                    vars: JSON.stringify(func.vars),
```

Comment:
I'm not sure if this is really needed because `func.vars` comes from the config file which should already be a JSON object.